src/document.h

```diff
@@ -2318,8 +2317,18 @@ struct Document {
         searchfilter = false;
         scrolltoselection = true;
         CollectCells(rootgrid);
-        loopv(i, itercells) itercells[i]->text.filtered =

```

Comment:
this particular change now needs and extra `sort`, calls to `lower_bound` and `upper_bound` and generally a lot more complex code.. why?